src/styles/blocks/card.css

```diff
@@ -0,0 +1,103 @@
+/* #region card */
+html {
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  box-sizing: border-box;
```

Comment:
Move it to * selector

Reply:
all these 4 components should I move in 0ne selector?